<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetRoomDialog</class>
 <widget class="QDialog" name="TargetRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房间号输入</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>房间号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_room"/>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>“，”或“ ”分割多
个房间号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_add">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
